apps/desktop/src/components/exporting/utils/svg-generator.ts

```
@@ -375,9 +453,16 @@ export const generateDrillChartExportSVGs = async (args: {
                 );
             else outputSVGs[0].push(canvas.toSVG());
         }
+
+        // Remove props so they can be recreated with next page's geometry
+        propObjects.forEach((obj) => canvas.remove(obj));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)